pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "mongomock"
+dynamic = ["version"]
+description = "Fake pymongo stub for testing simple MongoDB-dependent code"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+license = { file = "LICENSE" }
+authors = [
+    { name = "Rotem Yaari", email = "vmalloc@gmail.com" },
+    { name = "Martin Domke", email = "mail@martindomke.net" },
+]
```

Comment:
Please add myself in there `{ name = "Pascal Corpet", email = "pascal@corpet.net" },`